Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -696,6 +685,17 @@ private void LogEntryDeletion(OperationContext context, ShortHash hash, ContentL
         /// </returns>
         public abstract IEnumerable<StructResult<StrongFingerprint>> EnumerateStrongFingerprints(OperationContext context);
 
+        /// <summary>
+        /// Perform garbage collection of metadata entries.
+        /// </summary>
+        /// <param name="context">
+        ///     Tracing context.
+        /// </param>
+        /// <returns>
+        ///     Result providing the call's completion status.

```

Comment:
Should it be Result<T>? BoolResult or something similar?

Reply:
Used to,  but was not needed since we trigger this from a timer. Forgot to remove the comment.

---
In reply to: [309381229](https://github.com/microsoft/BuildXL/pull/679#discussion_r309381229) [](ancestors = 309381229)